packages/frontend/src/components/AppBar/index.tsx

```diff
@@ -47,7 +47,7 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
     <MuiAppBar
       color="transparent"
       position={drawerOpen && matchSmallScreens ? 'fixed' : 'relative'}
-      sx={{ zIndex: 1 }}
+      sx={{ zIndex: 11 }}
```

Comment:
v small thing but can i ask why you chose this number for the zindex ah

Reply:
it has to be higher than drawer